library/std/src/thread/local/tests.rs

```diff
@@ -258,7 +258,7 @@ fn join_orders_after_tls_destructors() {
                                 THREAD2_JOINED => panic!(
                                     "Thread 1 still running after thread 2 joined on thread 1"
                                 ),
-                                v => unreachable!("sync state: {}", v),
+                                v => unreachable!("sync state: {v}"),
```

Comment:
`unreachable!` was broken until recently https://github.com/rust-lang/rust/pull/93179 when there was only one argument given. This fix is in nightly but not yet to beta and stable and because std stage 0 use beta this won't work.
I would therefor advise to not yet update any `unreachable!` macro call.

Reply:
Thank you for the insight!